octobot_trading/exchanges/implementations/spot_exchange_simulator.py

```diff
@@ -15,10 +15,14 @@
 #  License along with this library.
 import octobot_trading.exchanges.connectors as exchange_connectors
 import octobot_trading.exchanges.types as exchanges_types
+from octobot_trading.exchanges.config import ccxt_exchange_settings
 
 
 #TODO remove
 class SpotExchangeSimulator(exchanges_types.SpotExchange):
+    CONNECTOR_CONFIG: ccxt_exchange_settings.CCXTExchangeConfig = (
+        ccxt_exchange_settings.CCXTExchangeConfig
```

Comment:
Same here (I don't understand why we have ccxt stuff in the generic margin exchange implementation)

Reply:
removed